native/android/app/src/main/res/values-ar/strings.xml

```diff
@@ -1,3 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I think these files are important. Else the app is not officially supporting 'ar' for example.

Reply:
Where is this relevant? I.e. where is it shown which languages are supported?